doc/workday/signature-suite.md

```diff
@@ -0,0 +1,81 @@
+# Workday Signature Suite
+- Authors: 
+  - Gabe Cohen [gabe.cohen@workday.com](mailto:gabe.cohen@workday.com)
+- Last updated: 2020-04-06
+
+## Status
+- Status: **DRAFT**
+- Status Date: 2019-02-20
+
+## Abstract
+Operating within the ecosystem of DIDs, Verifiable Credentials, and Distributed Ledger technology, Workday has a vested interest in a consistent, reliable, and secure method to sign over and verify different types of information. Additionally, we have designed the Workday Credentials system to be interoperable with the wider community. In the interest of all listed above, we propose our addition to the [Linked Data Cryptographic Suite Registry](https://w3c-ccg.github.io/ld-cryptosuite-registry).
+
+## Contents
+  * [Abstract](#abstract)
+  * [Contents](#contents)
+  * [Specification](#specification)
+    - [ID](#id)
+    - [Type](#type)
+      + [Verification Key](#verification-key)
+    - [Canonicalization Algorithm](#canonicalization-algorithm)
+    - [Digest Algorithm](#digest-algorithm)
+    - [Signature Algorithm](#signature-algorithm)
+  * [Prior Art](#prior-art)
+  * [Implementation Notes](#implementation-notes)
+  * [References](#references)
+
+## Specification
+According to the [Linked Data Signatures](https://w3c-dvcg.github.io/ld-signatures/) document, a _Linked Data Signature_
+
+>  is designed to be easy to use by developers and therefore strives to minimize the amount of information one has to remember to generate a signature. 
+
+The document outlines five properties that are required to comprise a Signature Suite: **id**, **type**, **canonicalizationAlgorithm**, **digestAlgorithm**, and **signatureAlgorithm**.
+
+This specification is intended to work only on JSON data.
```

Comment:
You might want to rephrase this as "This specification is tended to not rely on JSON-LD canonicalization"... JSON-LD is JSON.

Reply:
Thanks, will rephrase